cmd/agent/app/integrations.go

```diff
@@ -115,6 +121,59 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func detectEnvironment() error {
```

Comment:
Could we rename this function. It's difficult to understand what it does just by the name. Especially since it doesn't detect anything but check that the file are present.

Reply:
It checks 2 files and loads conf for 1 setting. What do you recommend renaming it to?

Reply:
Renamed!